src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("1.3.1.1514")]
+[assembly: AssemblyVersion("1.3.1.1977")]
```

Comment:
Dug around...can't figure out why this file was included, it's in the [`.gitignore` ](https://github.com/DynamoDS/Dynamo/blob/master/.gitignore#L74) for the repo, which is what my cloned version uses too.
Thanks for catching it, I'll exclude from future PRs.